SimulationOutputReport/measure.rb

```diff
@@ -764,13 +761,6 @@ def get_outputs(timeseries_frequency,
   end
 
   def check_for_errors(runner, outputs)
-    all_total = @fuels.values.map { |x| x.annual_output }.inject(:+)
```

Comment:
There're two check/test removed because we now only retain the unexpected unmet load. Is it ok to remove them? Or any suggestion?
I think as long as we have the unexpected load available, should we add test to set up limits of unexpected unmet loads?

Reply:
Addressed.